src/terminal/input/terminalInput.cpp

```diff
@@ -598,6 +598,13 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
         return true;
     }
 
+    // Check any other key mappings (like those for the F1-F12 keys).
+    const auto mapping = _getKeyMapping(keyEvent, _ansiMode, _cursorApplicationMode, _keypadApplicationMode);
+    if (_translateDefaultMapping(keyEvent, mapping, senderFunc))
+    {
+        return true;
+    }
+

```

Comment:
Since the Ctrl-key handling block below now handles many more combinations I decided to move this block of code upwards. This should prevent any accidental key mapping conflicts.